nifi/collectors/processgroups.go

```diff
@@ -30,111 +30,112 @@ type ProcessGroupsCollector struct {
 
 func NewProcessGroupsCollector(api *client.Client, labels map[string]string) *ProcessGroupsCollector {
 	prefix := MetricNamePrefix + "pg_"
-	statLabels := []string{"node_id", "group"}
+	//statLabels := []string{"node_id", "group"}
+	statLabelsId := []string{"node_id", "group", "entity_id"}
```

Comment:
What does `Id` mean in this context? If this is only added to contrast with the removed `statLabels`, then maybe it should just be `statLabels`?

Reply:
You're absolutely right. Fixed with new commits.